scratchie.js

```diff
@@ -179,9 +183,22 @@
             }
         }
 
+        if(e.pageX == undefined) {
```

Comment:
By changing `pageX` + `pageY` like the comments above you could remove the whole if conditions block :)